### ISSUE TITLE
Materiais de referência para workshop

### ISSUE THREAD

**Issue by mmsdocs** [GiovanniSM20/rust-roadshow-rj.rs#13]

**Comment by mmsdocs** [comment]
Existe muito [material disponível](https://www.rust-lang.org/en-US/documentation.html) sobre o ensino de Rust, inclusive [exercícios para oficinas](https://github.com/rust-community/rustbridge/tree/master/workshops/src), [exercícios para iniciantes](http://exercism.io/languages/rust/exercises) e até [coisas em português](https://blog.bltavares.com/). Nas oficinas Dive Into Rust realizados em 2017 no Brasil, utilizamos o material disponível em: [https://github.com/mozillabrasil/dive-into-rust](https://github.com/mozillabrasil/dive-into-rust). Este material está em constante evolução, fique a vontade para utilizar, modificar, propor mudanças ou desenvolver sua própria versão do material.

**Comment by GiovanniSM20** [comment]
Certo, estarei criando uma divisão dentro do repositório para me situar no material!
